schema/resource.go

```diff
@@ -138,6 +139,24 @@ func (r *Resource) Validate() error {
 	return nil
 }
 
+func (r *Resource) PrimaryKeyValue() *string {
+	pkColumns := r.Table.PrimaryKeys()
+	if len(pkColumns) == 0 {
+		return nil
+	}
+
+	parts := make([]string, 0, len(pkColumns))
+	for i, col := range r.Table.Columns {
+		if !col.CreationOptions.PrimaryKey {
+			continue
+		}
+		parts = append(parts, r.data[i].String())
+	}
+
+	res := "(" + strings.Join(parts, ",") + ")"
+	return &res
+}
```

Comment:
Because this is going to be part of the public interface, maybe we should just return `[]any` and then do the string concatenation where you need it?

Reply:
Yes, thought about it, too.

Reply:
ret `CQTypes`